src/templates/HelpArticle.jsx

```diff
@@ -8,6 +8,13 @@ import ArticlesList from 'components/ArticlesList'
 import QuickNav from 'components/QuickNav'
 import index from 'data/algolia.js'
 
+const HELP_DRAWER_COOKIE = 'help_drawer_cookie'
+const DRAWER_CLASSNAME = 'drawer'
+const MESSAGES = {
+  HELP_DRAWER_READY: 'drawer_ready',
+  HELP_DRAWER_REQUEST: 'drawer_request',
+}
+
 class HelpPageTemplate extends React.Component {
```

Comment:
I think the main help page (/help/) uses a different component. See `src/pages/help/Index.jsx`.

Reply:
Then I might do this with an HOC.

Reply:
But, let's not have this as a blocker - currently there are 2 `HelpTriggers` pointing to HelpArticles only.

Reply:
And when they navigate to /help/ the cached value will make sure they still see no footer/header, rght?

Reply:
Right